ParabolInc/parabol
# fix(docker): do not expose ports to the local network

How to test:

- Checkout to master
- Run yarn dev
- Figure out your laptop IP in the local network
- Using this IP and device connected to the same network access http://YOUR_IP:8080/, see that it is accessible
- Checkout to this branch (fix-docker-ports)
- **Warning! This might wipe local database data, I was not able to find alternative way to do this**: Run `docker-compose down` `docker-compose up -d`
- Run `docker ps` see that containers now have `127.0.0.1` address specified in the PORTS column
- Using this IP and device connected to the same network access http://YOUR_IP:8080/, see that it is not accessible anymore
- Check other ports too
